### ISSUE TITLE
Cannot disable hourly backups via automation editor

### ISSUE THREAD

**Issue by lloyda** [jcwillox/hass-auto-backup#73]
I had enabled (3) hourly backups via the Blueprint, but decided it was not really needed.   However if I try and stop it using the automation editor, the control makes no difference - I have to edit the automations config file.   v1.3.0

**Comment by jcwillox** [comment]
Yeah, I've noticed this occasionally as well, I believe it's an issue with Home Assistant, I don't believe there's anything I can do with the blueprint to fix that. If you encounter this I recommend switching to "Edit in YAML", and manually setting the option to `false` or `true`.
